themes/palenight-italic.json

```diff
@@ -1264,6 +1306,13 @@
         "foreground": "#82b1ff"
       }
     },
+    {
+      "name": "Asciidoc Function",
```

Comment:
Can you share a snippet of code?

Reply:
I use this [full sample file](https://gist.githubusercontent.com/mojavelinux/4402633/raw/f83ffb1049a7963039aac3db24f545d299540291/test.asciidoc)

There is a lot ot of function occurence in it, for example all the following orange keywords are considered "function" by the Asciidoc extension:
![image](https://user-images.githubusercontent.com/763200/93849999-8474a480-fcad-11ea-83e2-0e0b72fb81b7.png)